src/main/java/com/ioet/bpm/peopletime/timeevents/repositories/CustomRepositoryImpl.java

```diff
@@ -0,0 +1,40 @@
+package com.ioet.bpm.peopletime.timeevents.repositories;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBQueryExpression;
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+import com.ioet.bpm.peopletime.timeevents.domain.TimeEvent;
+
+import java.time.Instant;
+import java.util.*;
+
+public class CustomRepositoryImpl implements CustomRepository {
+
+    @Override
+    public Optional<TimeEvent> lastActiveTimeEvent(String personId) {
```

Comment:
Do we have tests for this class?

Reply:
Not for this class. But the method behavior is tested with the controller tests.